### ISSUE TITLE
Missing ability to set different highlight color for stacked bar chart?

### ISSUE THREAD

**Issue by liuxuan30** [danielgindi/ios-charts#246]
Today I met a problem that I want to set highlight colors for stacked bar chart. Each portion of the stacked bar chart can obtain its color via dataSet.colors, however, I don't see there is something like highlightColors for stacked bar chart. Seems it only uses highlightColor itself. I searched through v2.1.2 code, and still not seeing it. 

Do I missed something or we don't support it yet?

**Comment by danielgindi** [comment]
No matter what you do, they always want more customizations ;-)

**Comment by liuxuan30** [comment]
Got it. Just feeling it 'should' be there already :-) I will try to come up with one after I finish what's still on my plate. I notice you use highlighters now. that needs some time for me to upgrade my library too.

**Comment by danielgindi** [comment]
Yes but the Highlight class was moved as it is, not much changed there. The
code that handles the calculations regarding highlights was restructured
and improved, as it became too complex when we added stacked bars...

It would be great if you made a PR :-)

**Comment by liuxuan30** [comment]
I have made lots of customizations about highlights based on v2.1.0, so, I need to merge carefully and absorb the new change. Anyway, I can try first.

**Comment by Mel91** [comment]
I need this feature and wonder if it is implemented in the latest release of the library? I see that the issue is still open so I assume that it´s not implemented yet, just wanted a confirmation? If the library doesn't support this, then I wonder if anyone has any suggestion on how to work around this @liuxuan30  how did you solve this problem?

**Comment by liuxuan30** [comment]
It's not implemented yet. But you can easily add new properties for highlight colors, and override `drawaHighlighted`
```swift
                context.setFillColor(set.highlightColor.cgColor)
```
just read your highlight colors array.
